全新安装，打开应用即闪退（DocumentProvider 创建失败导致崩溃）

### 摘要 / Summary

全新安装，打开即闪退

### 重现步骤 / Steps to Reproduce

1.全新安装应用
2.点击图标

### 预期行为 / Expected Behavior

可进入引导页面

### 日志 / Log

time: 1767591447049
msg: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String java.io.File.getParent()' on a null object reference
stacktrace: java.lang.RuntimeException: Unable to get provider com.osfans.trime.provider.RimeDataProvider
	at android.app.ActivityThread.installProvider(ActivityThread.java:8741)
	at android.app.ActivityThread.installContentProviders(ActivityThread.java:8236)
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7897)
	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2584)
	at android.os.Handler.dispatchMessage(Handler.java:110)
	at android.os.Looper.dispatchMessage(Looper.java:315)
	at android.os.Looper.loopOnce(Looper.java:251)
	at android.os.Looper.loop(Looper.java:349)
	at android.app.ActivityThread.main(ActivityThread.java:9053)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:596)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:929)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String java.io.File.getParent()' on a null object reference
	at com.osfans.trime.provider.RimeDataProvider.onCreate(r8-map-id-04ed48887dfbb219c71217602fe92a0c16aa487cced2ed207c269c571b51027c:12)
	at android.content.ContentProvider.attachInfo(ContentProvider.java:2686)
	at android.content.ContentProvider.attachInfo(ContentProvider.java:2655)
	at android.provider.DocumentsProvider.attachInfo(DocumentsProvider.java:185)
	at android.app.ActivityThread.installProvider(ActivityThread.java:8738)
	... 12 more

### 截图 / Screenshot

[N/A]

### 附加信息 / Additional Context

全新安装应用，无载入方案，或使用万象内置方案版应用，都会闪退

### 系统版本 / OS Version

Android 16 QPR1 (CrDroid12.5)

### 应用版本 / App Version

v3.3.8-0-gf3f5c923-arm64-v8a (以及最新Nightly v3.3.8-8)

### 构建信息 / Build Information

无法进入主界面，无法复制

--- a/app/src/main/java/com/osfans/trime/provider/RimeDataProvider.kt
+++ b/app/src/main/java/com/osfans/trime/provider/RimeDataProvider.kt
@@ -72,7 +72,7 @@ class RimeDataProvider : DocumentsProvider() {
     private fun fileFromDocId(docId: String) = File(docIdPrefix, docId)
 
     override fun onCreate(): Boolean {
-        baseDir = context!!.getExternalFilesDir(null)!!
+        baseDir = context!!.getExternalFilesDir(null) ?: return false
         docIdPrefix = "${baseDir.parent}${File.separator}"
         textFilePaths = Array(TEXT_FILES.size) { baseDir.resolve(TEXT_FILES[it]).absolutePath }
         return true

